models/user.go

```diff
@@ -324,12 +324,12 @@ func (u *User) RelAvatarLink() string {
 
 	switch {
 	case u.UseCustomAvatar:
-		if !com.IsExist(u.CustomAvatarPath()) {
+		if !com.IsExist(u.CustomAvatarPath()) || !com.IsFile(u.CustomAvatarPath()) {
```

Comment:
Can just have `!com.IsFile(u.CustomAvatarPath())`; `com.IsFile(..)` will return `false` if the file does not exist.